src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -44,7 +45,13 @@
  *
  */
 public abstract class CommandLineProgram {
-    protected final Logger logger = LogManager.getLogger(this.getClass());
+    protected transient Logger logger = LogManager.getLogger(this.getClass());
+
+    private void readObject(java.io.ObjectInputStream in)
+            throws IOException, ClassNotFoundException {
+        in.defaultReadObject();
+        logger = LogManager.getLogger(this.getClass()); // Logger is not serializable (even by Kryo)
```

Comment:
Yes, it looks like that would fix it. That change can be handled in #2176 if folks agree about moving to SLF4J (I would support it).